src/login/containers/loginContainer.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import { connect } from 'react-redux';
+
+import "../css/login.css";
+import * as actionBuilder from "../actions/loginActionsBuilder";
+import FormComponent from '../components/loginFormComponent';
+
+class LoginView extends Component {
+
+  state = {
+    username: '',
+    password: '',
+  };
+
+  render() {
+    return (
+      <div className="login-view">
+        <div className="login-content">
+          <div className="login-wrap">
```

Comment:
Avoid this type of unwanted HTML hierarchy inclusion just to copy styles from the existing app, make sure we update the CSS rules according to the new hierarchy structure.